lib/numorstring/port.go

```diff
@@ -50,13 +51,16 @@ func PortFromString(s string) (Port, error) {
 
 	parts := strings.Split(s, ":")
 	if len(parts) != 2 {
-		return Port{}, errors.New("invalid port format")
+		msg := fmt.Sprintf("invalid port format (%s)", s)
+		return Port{}, errors.New(msg)
```

Comment:
just curious why can't the error message be inlined? like `errors.New(fmt.Sprintf("invalid port format (%s)", s))`